core/src/main/java/nl/nn/adapterframework/configuration/digester/DigesterRuleBase.java

```diff
@@ -65,12 +68,14 @@ protected String getObjectName() {
 	}
 
 	/**
-	 * Add a configuration warning message to the current configuration
+	 * Add a configuration warning message to the current configuration.
+	 * Confirm {@link IbisException} when the cause is a {@link SAXParseException}.
```

Comment:
Wat bedoel je hier?

Reply:
dat het `line [xxx] column [xxx]` moet zijn en niet `line xxx col xxx`

Reply:
Maar hoezo confirm je een IbisException?
Of bedoel je 'conform', zoals in 'display location conform IbisException when the cause is a SAXParseException'?